easybuild/easyconfigs/l/LibUUID/LibUUID-1.0.3-GCCcore-6.4.0.eb

```diff
@@ -9,14 +9,25 @@ description = """Portable uuid C library"""
 
 toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
 
-source_urls = [SOURCEFORGE_SOURCE]
-sources = [SOURCELOWER_TAR_GZ]
-checksums = ['46af3275291091009ad7f1b899de3d0cea0252737550e7919d17237997db5644']
+source_urls = [
+    SOURCEFORGE_SOURCE,
+    'https://raw.githubusercontent.com/karelzak/util-linux/master/libuuid/src',
```

Comment:
@ocaisa Can we avoid downloading from `master`, and use a particular version/tag instead, since this is effectively a moving target...